forge-gui/res/cardsfolder/upcoming/mana_cannons.txt

```diff
@@ -0,0 +1,8 @@
+Name:Mana Cannons
+ManaCost:2 R
+Types:Enchantment
+T:Mode$ SpellCast | ValidCard$ Card.MultiColor | ValidActivatingPlayer$ You | Execute$ TrigDamage | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a multicolored spell, CARDNAME deals X damage to any target, where X is the number of colors that spell is.
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ X
+SVar:X:TriggeredCard$CardNumColors
+SVar:BuffedBy:Card.MultiColor
+Oracle:Whenever you cast a multicolored spell, Mana Cannons deals X damage to any target, where X is the number of colors that spell is.
```

Comment:
AI:RemoveDeck:Random - the AI shouldn't put this in a mono red deck... but we don't have a more specific way to tell it that yet™️

Reply:
> yet™️

Ooh the foreshadowing  ;)